src/revolve/source_template/mongodb/service.py

```
@@ -0,0 +1,53 @@
+def mongo_serial(obj):
+    if isinstance(obj, (datetime, date)):
+        return obj.isoformat()
+    if isinstance(obj, ObjectId):
+        return str(obj)
+    return obj
+
+class HelloMongoResource:
+    def on_get(self, req, resp):
+        _test_mode = req.get_header('X-Test-Request') == 'true'
+        try:
+            client = get_db_connection()
```

Pylint (3.3.7)

[error] No value for argument 'test_mode' in function call

(E1120)